lib/ramble/ramble/cmd/workspace.py

```diff
@@ -51,6 +51,7 @@
     "info",
     "edit",
     "mirror",
+    "experiment-logs",
```

Comment:
May be for later, but do you think there's value to make this subcommand look for logs from other pipelines (setup, archive, etc.)?